plaso/parsers/wincc.py

```diff
@@ -0,0 +1,362 @@
+# -*- coding: utf-8 -*-
+"""Text parser plugin for WinCC log files."""
+
+from dfvfs.helpers import text_file
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class SIMATICS7EventData(events.EventData):
+  """SIMATIC S7 event data.
+
+  Attributes:
+    creation_time (dfdatetime.DateTimeValues): date and time the log entry
+        was created.
+    message (str): the message of the event.
```

Comment:
message has a special purpose in Plaso (see: https://plaso.readthedocs.io/en/latest/sources/user/Output-and-formatting.html#dynamic-output-module-fields) use body (or equiv instead)

Reply:
done